ignite/pkg/cosmosgen/generate.go

```diff
@@ -128,69 +127,6 @@ func (g *generator) setup() (err error) {
 	return nil
 }
 
-func (g *generator) resolveDependencyInclude() ([]string, error) {
```

Comment:
Will changing the behavior of cosmosgen like this break the ts client generation logic? Or we are able to keep it working with the buf version?

Reply:
We are able to keep it working with this 3trd party buf plugin: https://buf.build/community/stephenh-ts-proto

The `buf.gen.ts.yaml` is already in this PR:
https://github.com/ignite/cli/blob/6c10015825041bdeeee2cfe0769919413008982f/ignite/templates/app/files/proto/buf.gen.ts.yaml

Reply:
Are you sure? We generate a custom client with a custom template and it depends on resolving 3rd party dependencies from go.mod.